addons/skin.confluence/720p/DialogContentSettings.xml

```diff
@@ -60,158 +58,20 @@
 				<textcolor>white</textcolor>
 				<shadowcolor>black</shadowcolor>
 			</control>
-			<control type="label">
-				<description>Used Scraper Header</description>
-				<left>30</left>
-				<top>150</top>
-				<width>320</width>
-				<height>20</height>
-				<font>font12</font>
-				<label>$LOCALIZE[31312]</label>
```

Comment:
I'm not sure what I had there in my tests. But sure, we can do that, if you would like it.